regular-expressions/introduction/intro-to-regular-expressions/regex-tools-for-testing-regex.md

```diff
@@ -38,7 +38,7 @@ This is the best site for any mathematicians as it digs a little deeper into the
 
 ## Practice
 
-Once you've learnt some regex syntax it's recommended to test out your Regex patterns by using ???.
+Once you've learnt some syntax it's recommended to test out your patterns by using ???.
```

Comment:
let's use the american spelling here
```suggestion
Once you've learned some syntax, it's recommended to test out your patterns by using ???.
```

Reply:
I wasn't sure if I was supposed to change that because they are both correct English words. Grammarly wanted me to :D

Reply:
yeah, both are correct. learned is mostly used in america/canada while learnt is used in the UK. since our primary user base is American we should go with learned